graph.py

```diff
@@ -10,11 +12,54 @@
     log.error(f"Package {PACKAGE} not found. Please 'pip install {PACKAGE}'")
 
 
+class Filter:
+    """Class responsible for removing items with disqualifying values."""
+
+    def __init__(self, filter_type, filter_function):
+        self._filter_type = filter_type
+        self._filter_fun = filter_function
```

Comment:
Please avoid abbreviations. This reads more like "fun filter" than "filter function".